packages/build/tests/edge_functions/fixtures/functions_invalid/netlify/edge-functions/functions-1.ts

```diff
@@ -1,6 +1,2 @@
 // intentionally break function
-export async () => {
-  return new Response("Helloooooo, World!", {
-    headers: { "content-type": "text/html" },
-  })
-}
+export async () => 
```

Comment:
We're generating ESZIP instead of JS since https://github.com/netlify/edge-bundler/pull/224, and the ESZIP bundler has a different sensitivity for build time errors.